codegen/include/CodegenOptions.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef NNCC_CODEGENOPTIONS_H
+#define NNCC_CODEGENOPTIONS_H
+
+namespace luci_codegen
+{
+
+enum class ArchType
+{
+  Native,
```

Comment:
Is `Native` same as build machone architecutre? This seems implecit type and what is the needs or benifits of providing `Native`? As I understand, we're targeting ARM32 or AARCH64...

Reply:
> Is Native same as build machine architecutre?

Yes. I want to have this option as a default to simplify debugging/prototyping.

Halide looks stable, so I assume that most of bugs could be found using any target architecture. And debugging on the same machine is easier.